packages/platforms/android-player/src/ts/classes/android-machine.ts

```diff
@@ -67,7 +67,7 @@ export class AndroidMachine implements Machine {
     let appInfo;
     try {
       appInfo = await App.getInfo();
-    } catch (e) {
+    } catch {
```

Comment:
wouldn't it be better to cast the e argument to Error and throw that instead?

Reply:
I think this was written like this to make the code run without the App plugin (when running the code locally in a web browser). This is handy during development since it saves the overhead of building the Android app every time something in the ts code is changed. I also didn't want to change any functionality in this PR since it is just about linting.